assets/src/design-system/components/toggle/index.js

```diff
@@ -35,9 +35,9 @@ const CIRCLE_FINAL_POSITION =
   CIRCLE_INITIAL_POSITION -
   (CIRCLE_DIAMETER / 2 - CIRCLE_INITIAL_POSITION / 2);
 
-const ICON_WIDTH = 7;
-const ICON_TOP_POSITION = 8;
-const ICON_LEFT_POSITION = 30.5;
+const ICON_WIDTH = 18;
+const ICON_TOP_POSITION = 2;
+const ICON_LEFT_POSITION = 26;
```

Comment:
💭 do you like the checkmark in the current position or up 1 pixel?

![checkmark-position](https://user-images.githubusercontent.com/22185279/107675797-adde9e80-6c55-11eb-9e97-fb5b0faa4e4b.gif)

Reply:
To me it feels like it's better centered when it's slightly higher (maybe even 0.5 px), but I'll leave that to your judgement

Reply:
Great catch! I've forwarded the question to UX.

Reply:
Can we move it one pixel up. looks more centered.

Reply:
Fixed in e08c625c6.